CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_snapchat_ads v0.4.0
```

Comment:
Same comment from the source. Can we add the model renaming (both renaming from the source and transform packages but especially the transform renames) and the variable name changes from the source package as breaking changes to this entry as well.